cibuildwheel/platforms/android.py

```diff
@@ -617,8 +617,8 @@ def test_wheel(state: BuildState, wheel: Path) -> None:
 
     # Parse test-command.
     test_args = shlex.split(test_command)
-    if test_args[:2] in [["python", "-c"], ["python", "-m"]]:
-        test_args[:3] = [test_args[1], test_args[2], "--"]
+    if test_args[0] in ["python", "python3"] and any(arg in test_args for arg in ["-c", "-m"]):
+        del test_args[0]
```

Comment:
This code looks a little odd, let's comment that.
```suggestion
        # Pass the args to be parsed by the cpython testbed script. We only do this
        # when '-c' or '-m' is found in the command, because without those flags, the
        # testbed script will prepend '-m test', which isn't what the user wants.
        del test_args[0]
```

Reply:
Thanks; I've added the comment and rephrased it slightly.

Reply:
thanks!